security-council-rehearsals/templates/r1-hello-council/SignFromJson.s.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.15;
+
+// This import path should be relative to where the rehearsal script will be located after a new rehearsal is setup,
+// which is one level above the current location in
+// repo_root/security-council-rehearsals/<rehearsal-dir>/SignFromJson.s.sol
+import {SignFromJson as OriginalSignFromJson} from "../../script/SignFromJson.s.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
```

Comment:
This import shouldn't be necessary, but it [wasn't compiling](https://app.circleci.com/pipelines/github/ethereum-optimism/superchain-ops/1632/workflows/5d6e7f19-24ea-4987-819c-1abe15a3c4f0/jobs/14890) without it, and who am I to blow against the wind.

Reply:
I've noticed this before too and was surprised the import was needed, it's harmless enough at least 🤷